src/pages/ApiParticulier.js

```diff
@@ -82,24 +95,63 @@ const DonneesDescription = () => (
   </section>
 );
 
+const donneesDisponibles = [
```

Comment:
it is not clear here that we are dealing with scopes (referring to the [oauth2 nomenclature](https://tools.ietf.org/html/rfc6749#section-3.3))

Reply:
I renamed it `availableScopes`